### PR TITLE
[docs] add `app_path` input to `eas/maestro` function group schema

### PR DESCRIPTION
**Pull request by szdziedzic**
# Why

Add `app_path` input to `eas/maestro` function group schema

# How

Add `app_path` input to `eas/maestro` function group schema

# Test Plan

Preview

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by sjchmiela — APPROVED** [review]

File: docs/pages/custom-builds/schema.mdx [hunk 1]
```diff
@@ -740,9 +740,10 @@ You can replace the `eas/build` command call by using these steps in your YAML c
...
-| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run. |
+| Input       | Description                                                                                                                                                                                                                   |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `flow_path` | **string** Path (or multiple paths, each in a separate line) to [Maestro flows](https://maestro.mobile.dev/getting-started/writing-your-first-flow) to run.                                                                   |
+| `app_path`  | **string** Path (or regex pattern) to the simulator/emulator app that should be tested. If not provided defaults to `ios/build/Build/Products/*simulator/*.app` for iOS and `android/app/build/outputs/**/*.apk` for Android. |
```
Comment (Simek) [inline docs/pages/custom-builds/schema.mdx:746]:
```suggestion
| `app_path`  | `string` Path (or regex pattern) to the emulator/simulator app that should be tested. If not provided defaults **android/app/build/outputs/**/*.apk** for Android and to **ios/build/Build/Products/*simulator/*.app** for iOS. |
```

Android before iOS + formatting nits. 

Would be nice to move the type into separate column in the future, but this improvement is definitely out of this PR scope.

**Review by Simek — APPROVED** [review]

File: docs/pages/custom-builds/schema.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/custom-builds/schema.mdx:746]: I'll follow up to create a `type` separate column, thanks for the feedback @Simek!

We should probably keep `string` in bold text unless we decide to change it for both fields in this table (which might be out of scope of the PR).

**Review by amandeepmittal — APPROVED** [review]

File: docs/pages/custom-builds/schema.mdx (hunk 1 above)
Comment (amandeepmittal) [inline docs/pages/custom-builds/schema.mdx:746]:
```suggestion
| `app_path`  | **string** Path (or regex pattern) to the simulator/emulator app that should be tested. If not provided, it defaults to `ios/build/Build/Products/*simulator/*.app` for iOS and `android/app/build/outputs/**/*.apk` for Android. |
```

missing context about "what" defaults to the values. Please also see Simek's comment which provides suggestion on using the correct platform order as per our writing style guide.

Comment (szdziedzic) [reply docs/pages/custom-builds/schema.mdx:746]: >Would be nice to move the type into separate column in the future, but this improvement is definitely out of this PR scope.

Yeah, good idea

**Pull request merged**